src/pages/Selector/index.jsx

```diff
@@ -38,3 +39,7 @@ const Selector = () => (
 );
 
 export default container(Selector);
+
+Selector.propTypes = {
```

Comment:
Small stylistic thing but I personally prefer export to be the last line of the file. Just small convention things :)